msi/make.cmd

```diff
@@ -0,0 +1,33 @@
+rmdir /s/q build
+
+rem TODO: Where should the git-credential-winstore, gcm and posh-git binaries
+rem       be found/stored?
+
+rem TODO: Generate GitComponents.wxs from ..\make-file-list.sh
+rem
+rem       1. Files installed to `bin\` folder use Directory='BinFolder' and
+rem          all others use the pattern Directory='INSTALLFOLDER:\path\'.
+rem
+rem       2. The `git.exe` file in `BinFolder` is given the explicit
+rem          Id='GetExe' all other files are anonymous (do not have an Id).
+
+rem Build the .msi.
+rem
+rem TODO: Correctly set the bind paths (change the -b gitfileshere switch)
+rem       to the real source location for the binaries to include in the
+rem       installation package.
+rem
+rem TODO: Set the output paths appropriately.
+rem
+wix\candle.exe GitProduct.wxs GitComponents.wxs GitCredStoreComponents.wxs GitPoshComponents.wxs -o build\obj\  -ext WixUtilExtension
+wix\light.exe build\obj\GitProduct.wixobj build\obj\GitComponents.wixobj build\obj\GitCredStoreComponents.wixobj build\obj\GitPoshComponents.wixobj -o build\msi\git.msi -ext WixUtilExtension -b gitfileshere -b ..\installer -sval
+
+rem Build the bundle.
```

Comment:
The msi is the MSI package. The bundle is an executable, often called a bootstrapper, that can wrap the .msi or the .msi can be external (but when shipping over the internet single executables--under 200MB--are easier to distribute). The bundle does work to cache the .msi package, register a stable source and several other things that improve the installation success rate of installing .msi packages over the internet.